lib/features/inlinedIcon/index.jsx

```diff
@@ -0,0 +1,176 @@
+          </svg>
+        )
+
+      case INLINED_ICONS.QUESTION:
+        return (
+          <svg className={styles.icon} viewBox='0 0 20 20'>
+            <g transform='translate(2.000000, 2.000000)'>
+              <path
+                d='M8,16 C12.418278,16 16,12.418278 16,8 C16,3.581722 12.418278,0 8,0 C3.581722,0 0,3.581722 0,8 C0,12.418278 3.581722,16 8,16 L8,16 Z M8,15 C4.13400675,15 1,11.8659932 1,8 C1,4.13400675 4.13400675,1 8,1 C11.8659932,1 15,4.13400675 15,8 C15,11.8659932 11.8659932,15 8,15 L8,15 Z'
+                className={styles.fill}></path>
+              <path
+                d='M8.5,9.54003906 C8.5,9.25177955 8.61326781,9.11890691 9.06315953,8.85553321 C8.97875302,8.90494611 9.37878081,8.67380503 9.48395355,8.60897505 C9.72920228,8.45780023 9.92654737,8.31148899 10.1066093,8.13591664 C10.8475673,7.41343344 11,6.98233157 11,6 C11,4.34314575 9.65685425,3 8,3 C6.34314575,3 5,4.34314575 5,6 C5,6.27614237 5.22385763,6.5 5.5,6.5 C5.77614237,6.5 6,6.27614237 6,6 C6,4.8954305 6.8954305,4 8,4 C9.1045695,4 10,4.8954305 10,6 C10,6.73058436 9.93924203,6.90241678 9.40848485,7.41994024 C9.28805746,7.53736493 9.14689108,7.64202538 8.9592205,7.75770822 C8.86825969,7.81377776 8.48665124,8.03427591 8.55794825,7.99253752 C7.8332105,8.41681042 7.5,8.80769425 7.5,9.54003906 C7.5,9.81618144 7.72385763,10.0400391 8,10.0400391 C8.27614237,10.0400391 8.5,9.81618144 8.5,9.54003906 L8.5,9.54003906 Z'
+                className={styles.fill}></path>
+              <path
+                d='M8,13 C8.27614237,13 8.5,12.7761424 8.5,12.5 C8.5,12.2238576 8.27614237,12 8,12 C7.72385763,12 7.5,12.2238576 7.5,12.5 C7.5,12.7761424 7.72385763,13 8,13 Z'
+                className={styles.fill}></path>
+              <path
+                d='M8,10.5 L8,9.5'
+                className={styles.stroke}
+                strokeLinecap='round'></path>
+            </g>
+          </svg>
+        )
+    }
+  }
+}
+
+renderInlinedIcon.defaultProps = {
+  icon: undefined
+}
+
+renderInlinedIcon.propTypes = {
+  icon: PropTypes.oneOf(
+    Object.keys(INLINED_ICONS).map((key) => INLINED_ICONS[key])
```

Comment:
You're doing this in many places, I guess it deserve a helper on it's own (`lib/object/values`)